### PR TITLE
Load private projects into dw

### PR DESCRIPTION
**Pull request by gondzo**
@RishiRajSahu before executing, need to create a table and view in dw with these queries

```
CREATE TABLE private_project (
 project_id DECIMAL(12,0),
 component_id DECIMAL(12,0),
 component_name NVARCHAR(128),
 num_registrations DECIMAL(5,0),
 num_submissions DECIMAL(5,0),
 num_valid_submissions DECIMAL(5,0),
 avg_raw_score DECIMAL(5,2),
 avg_final_score DECIMAL(5,2),
 phase_id DECIMAL(5,0),
 phase_desc NVARCHAR(64),
 category_id DECIMAL(12,0),
 category_desc NVARCHAR(64),
 posting_date DATETIME YEAR TO SECOND, 
 submitby_date DATETIME YEAR TO SECOND, 
 complete_date DATETIME YEAR TO SECOND, 
 review_phase_id DECIMAL(3,0),
 review_phase_name NVARCHAR(30),
 status_id DECIMAL(3,0),
 status_desc NVARCHAR(50),
 level_id DECIMAL(5,0),
 version_id DECIMAL(3,0),
 version_text NCHAR(20),
 rating_date DATETIME YEAR TO FRACTION(3),
 viewable_category_ind DECIMAL(1,0),
 num_submissions_passed_review DECIMAL(5,0),
 winner_id DECIMAL(10,0),
 stage_id DECIMAL(6,0),
 digital_run_ind DECIMAL(1,0) DEFAULT 1, 
 suspended_ind DECIMAL(1,0) DEFAULT 0, 
 project_category_id INTEGER, 
 project_category_name VARCHAR(254),
 tc_direct_project_id INTEGER, admin_fee DECIMAL(10,2),
 contest_prizes_total DECIMAL(10,2),
 client_project_id INTEGER, 
 start_date_calendar_id DECIMAL(12,0),
 duration DECIMAL(10,2),
 fulfillment DECIMAL(10,2),
 last_modification_date DATETIME YEAR TO SECOND, 
 first_place_prize DECIMAL(10,2),
 num_checkpoint_submissions DECIMAL(5,0),
 num_valid_checkpoint_submissions DECIMAL(5,0),
 total_prize DECIMAL(10,2),
 checkpoint_start_date DATETIME YEAR TO SECOND, 
 checkpoint_end_date DATETIME YEAR TO SECOND, 
 challenge_manager DECIMAL(10,0),
 challenge_creator DECIMAL(10,0),
 copilot DECIMAL(10,0),
 challenge_launcher DECIMAL(10,0),
 registration_end_date DATETIME YEAR TO SECOND, 
 scheduled_end_date DATETIME YEAR TO SECOND, 
 checkpoint_prize_amount DECIMAL(10,2),
 checkpoint_prize_number DECIMAL(5,0),
 dr_points DECIMAL(10,2),
 reliability_cost DECIMAL(10,2),
 review_cost DECIMAL(10,2),
 forum_id DECIMAL(10,0),
 submission_viewable DECIMAL(1,0),
 is_private DECIMAL(1,0),
 actual_total_prize DECIMAL(10,2),
 copilot_cost DECIMAL(10,2),
 estimated_reliability_cost DECIMAL(10,2),
 estimated_review_cost DECIMAL(10,2),
 estimated_copilot_cost DECIMAL(10,2),
 estimated_admin_fee DECIMAL(10,2),
 PRIMARY KEY (project_id) CONSTRAINT private_project_pkey, 
 FOREIGN KEY (stage_id) REFERENCES stage(stage_id) CONSTRAINT private_project_stage_fk, 
 FOREIGN KEY (client_project_id) REFERENCES client_project_dim(client_project_id) CONSTRAINT private_project_client_project_dim_id_fk
 );
```
and
```
create view all_project as
select * from project
union all
select * from private_project
```

### CODE REVIEW

**Comment by RishiRajSahu** [comment]
@gondzo yep agree could you execute those on your test DW instance and raise PR for code as well in parallel ?

File: src/main/com/topcoder/utilities/dwload/TCLoadTCS.java [hunk 1]
```diff
@@ -262,6 +262,15 @@
             " and p.project_id not in (select ce.contest_id from contest_eligibility ce " +
                     " where ce.is_studio = 0) ";
 
+    /**
+     * SQL fragment to be added to a where clause to select projects with eligibility constraints
+     *
+     * @since 1.1.5
+     */
+    private static final String WITH_ELIGIBILITY_CONSTRAINTS_SQL_FRAGMENT =
```
Comment (RishiRajSahu) [inline src/main/com/topcoder/utilities/dwload/TCLoadTCS.java]: @gondzo I think this constraint is similar to other constraint

**Comment by RishiRajSahu** [comment]
@ajefts this looks good to me to add private projects to DW and create a new view for supply-etl. Still would like you to review it once.

**Comment by ajefts** [comment]
@RishiRajSahu It looks ok to me in general.  One concern I have is the performance of the view.  Please also check the existing project table to see what indexes are on it and let's add the same indexes to the new table.  thanks.

**Comment by gondzo** [comment]
there are only default indexes on both tables (primary and foreign key columns), so nothing else to do here

**Comment by RishiRajSahu** [comment]
Created new table, indices, and view in dev instance of DW

**Pull request merged**